rpmlint/checks/FileDigestCheck.py

```diff
@@ -34,46 +38,90 @@ def _get_digest_group_from_path(self, path):
                     pass
         return None
 
-    def check_binary(self, pkg):
+    def _in_digest_configuration_group(self, pkgfile):
+        if stat.S_ISDIR(pkgfile.mode):
+            return False
+        return self._get_digest_location_from_path(pkgfile.name)
+
+    def _check_symlinks(self, pkg):
```

Comment:
It doesn't only check for symlinks but also for ghost files. Maybe name the function something more generic like `_check_filetypes()` or so.

Reply:
Yes.